py/makemoduledefs.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+# This pre-processor parses provided objects' c files for 
+# `MP_REGISTER_MODULE(module_name, obj_module, enabled_define)`
```

Comment:
I'm not sure that (markdown?) markup here makes sense. At least, we don't use it anywhere else.

Reply:
Sure, yeah, I do find myself doing that a bit these days...